src/dns-test.js

```diff
@@ -352,10 +351,9 @@ async function fetchTest() {
     // running if we don't get the response we expect. Also, if the fetch fails, 
     // it doesn't make sense to send anything to telemetry, because that 
     // may fail, too.
-    let response = await fetch("https://dnssec-experiment-moz.net/");
-    let responseOk = await response.ok;
+    let response = await fetch("https://dnssec-experiment-moz.net/", {cache: "reload"});
     let responseText = await response.text();
-    if (!(responseOk && responseText && responseText === "Hello, world!\n")) {
+    if (!(responseText === "Hello, world!\n")) {
```

Comment:
Use `!==` instead of `!( ... === ... )`. Logic is equivalent, but the former is more idiomatic.

Reply:
Addressed by 0990075